src/Render.cc

```diff
@@ -16,11 +18,15 @@ using namespace snowcrash;
 namespace drafter {
 
     // JSON content-type regex
-    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json";
+    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json[[:blank:]]*(;.*|$)";
```

Comment:
Why was this changed? Can we get some test that would fail without this changed?